homeassistant/components/camera/webrtc.py

```diff
@@ -78,7 +78,14 @@ class WebRTCAnswer(WebRTCMessage):
 class WebRTCCandidate(WebRTCMessage):
     """WebRTC candidate."""
 
-    candidate: str
+    candidate: RTCIceCandidate
+
+    def as_dict(self) -> dict[str, Any]:
+        """Return a dict representation of the message."""
+        return {
```

Comment:
This method will removed in a follow-up, where also the frontend will migrate to `RTCIceCandidate`